.github/workflows/build.yaml

```diff
@@ -68,3 +68,31 @@ jobs:
           RAILS_ENV: test
           RSPEC_RETRY_RETRY_COUNT: 3
         run: bundle exec rake
+  migrations:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: weg-li_test
+        ports: ["5432:5432"]
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
I suggest to update the checkout action for the "build" step as well. I can do it, but did not want to burden this PR with it @phoet. What do you think?

Reply:
Please go ahead, there were probably some unnecessary steps in here at time of writing when actions were quite new

Reply:
Same as https://github.com/weg-li/weg-li/pull/696#discussion_r1355060023